openml/datasets/functions.py

```diff
@@ -499,6 +500,8 @@ def get_dataset(
                 )
             except urllib3.exceptions.MaxRetryError:
                 parquet_file = None
+            if parquet_file is None and arff_file:
```

Comment:
What happens if both are None? Would that be an issue?

Reply:
No, that was already possible behavior (and is expected when lazy loading - though then this part of code won't be hit). Note that all I changed here is provide a warning.